### ISSUE TITLE
Dashboard tab panels with tables extend under output panel

### ISSUE THREAD

**Issue by Charles-Gagnon** [microsoft/azuredatastudio#10627]
This happens for the Agent and SQL Server Assessment extensions (the only ones which use this table in a dashboard I believe)

![Capture](https://user-images.githubusercontent.com/28519865/83281378-dfc5d880-a18c-11ea-8eda-60713dca22ec.gif)

**Comment by alanrenmsft** [comment]
this is because the panel resize event is not wired all the way to the agent control, to fix this, agent extension and assessment extension should listen to the mouse move event and update the size of the grid.

**Comment by Charles-Gagnon** [comment]
I would expect that the visible content area is controlled by the core dashboard code - not the extension. The size of the table within it can be controlled but the scrollbar should be for the content container - not the table. How are extensions supposed to know the size of the visible content area since they don't have access to the DOM?

**Comment by kburtram** [comment]
Yeah, it seems like dashboard needs to have a standard way to forward resize events to extensions, and extensions may need some logic to handle this.  Since both these extensions need to be migrated to modelview in the next several months, it may be better to make sure this type of scenario works well with that framework than over-invest in these custom UI extensions.

**Comment by aisbergde** [comment]
is there any workaround how to see the full list? Or is the only solution to change the sort order to see hidden elements?
